twitter/twitter_connector.bal

```diff
@@ -348,28 +312,31 @@ function TwitterConnector::getTopTrendsByPlace(int locationId) returns Trends[]|
     Trends[] trends = [];
     match httpResponse {
         error err => {
-            twitterError.message = err.message;
-            return twitterError;
+            return err;
         }
         http:Response response => {
             int statusCode = response.statusCode;
             var twitterJSONResponse = response.getJsonPayload();
             match twitterJSONResponse {
                 error err => {
-                    twitterError.message = err.message;
-                    return twitterError;
+                    return err;
                 }
                 json jsonResponse => {
                     if (statusCode == 200) {
                         trends = convertTrends(jsonResponse);
                         return trends;
                     } else {
-                        twitterError.message = jsonResponse.errors[0].message.toString();
-                        twitterError.statusCode = statusCode;
-                        return twitterError;
+                        return setResponseError(statusCode, jsonResponse);
                     }
                 }
             }
         }
     }
 }
+
+function setResponseError(int statusCode, json jsonResponse) returns error {
```

Comment:
Move this method to twitter_utils.bal file

Reply:
Fixed.